02-microarray/pathway-analysis_microarray_01_ora.Rmd

```diff
@@ -111,9 +113,12 @@ If the concept of a "file path" is unfamiliar to you; we recommend taking a look
 
 # Using a different refine.bio dataset with this analysis?
 
-If you'd like to adapt an example analysis to use a different dataset from [refine.bio](https://www.refine.bio/), we recommend placing the files in the `data/` directory you created and changing the filenames and paths in the notebook to match these files (we've put comments to signify where you would need to change the code).
-We suggest saving plots and results to `plots/` and `results/` directories, respectively, as these are automatically created by the notebook.
-From here you can customize this analysis example to fit your own scientific questions and preferences.
+If you'd like to adapt an example analysis to use a different dataset from [refine.bio](https://www.refine.bio/), we recommend replacing the `dge_url` with a different file path to a gene list that you are interested in. 
```

Comment:
We should specify what this file will contain: it is not just a gene list in this case, but a file with all the genes, each with p values and logFC. Not just any file will work in this code example.

I might add something about substituting the actual gene and background lists, rather than this specific file, with its specific format.

Reply:
Added some advice/caution about what needs to be changed.